src/main/java/uk/ac/ebi/eva/pipeline/io/VepProcess.java

```diff
@@ -168,7 +175,25 @@ public void flush() throws IOException {
         if (!isOpen()) {
             throw new IllegalStateException("Process must be initialized (hint: call open() before flush())");
         }
-        processStandardInput.flush();
+        tryWithTimeout(() -> {
+            processStandardInput.flush();
+            return null;
+        });
+    }
+
+    private void tryWithTimeout(Callable<Void> callable) {
```

Comment:
This method is placed a bit arbitrarily: it is not after its first usage nor is at the beginning of a set of private methods.